Automattic/wp-calypso
# Remove title attribute from header-nav

Related issue: https://github.com/Automattic/dotcom-forge/issues/2765

## Proposed Changes

We need to remove the title attribute from all logged out navigation links.

## Testing Instructions

* Confirm the title tags are not showing up anymore on: `/themes`, `/plugins`, `/tags`, `/support`
* Check the mobile menu
